pkg/pool/pool.go

```diff
@@ -322,6 +330,13 @@ func (p *pool) UpdatePool(job *worker.WarmPoolJob, didSucceed bool) (shouldRecon
 		log.Error(fmt.Errorf("warm pool job failed: %v", job), "operation failed")
 	}
 
+	if p.isPDPool {
+		p.prefixAvailable = prefixAvailable
```

Comment:
I think this is the only place we update this variable `prefixAvailable`, isn't it?

Reply:
Yes that's right.